sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2858,6 +2858,22 @@ object GpuOverrides extends Logging {
           GpuArrayRemove(lhs, rhs)
       }
     ),
+    expr[MapFromArrays](
+      "Creates a new map from two arrays",
+      ExprChecks.binaryProject(
+        TypeSig.MAP.nested(TypeSig.commonCudfTypes),
+        TypeSig.MAP.nested(TypeSig.commonCudfTypes),
+        ("array",
+          TypeSig.ARRAY.nested(TypeSig.commonCudfTypes),
+          TypeSig.all),
```

Comment:
Technically Spark does not support all types for the keys. I think it should be `TypeSig.all - TypeSig.MAP`